extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/servlet/WebAppServlet.java

```diff
@@ -144,7 +144,11 @@ public void service(ServletRequest req, ServletResponse res) throws ServletExcep
             if (widgetId == null || widgetId.isEmpty() || widgetId.equals(sitemapName)) {
                 // we are at the homepage, so we render the children of the sitemap root node
                 if (subscriptionId != null) {
-                    subscriptions.setPageId(subscriptionId, sitemap.getName(), sitemapName);
+                    if (subscriptions.exists(subscriptionId)) {
+                        subscriptions.setPageId(subscriptionId, sitemap.getName(), sitemapName);
+                    } else {
+                        logger.warn("Basic UI requested a non-existing event subscription id ({})", subscriptionId);

```

Comment:
Should we really fill the log with a warning because a user of the Basic UI requests a subscription ID that does not exist?
As it is a web page I assume a incorrect request is responded (logical) correctly without a message in the log.